Sources_v3/StreamChatUI/ChatChannelList/ChatChannelListItemView.swift

```diff
@@ -5,13 +5,15 @@
 import StreamChat
 import UIKit
 
-open class ChatChannelListItemView<ExtraData: ExtraDataTypes>: View, UIConfigProvider {
+open class ChatChannelListItemView<ExtraData: ExtraDataTypes>: View, UIConfigProvider, UIGestureRecognizerDelegate {
     override public func defaultAppearance() {
         backgroundColor = uiConfig.colorPalette.generalBackground
     }
 
     // MARK: - Properties
-    
+
+    public var trailingConstraint: NSLayoutConstraint?
+
     public var uiConfig: UIConfig<ExtraData>
```

Comment:
nit: would be nice to update this view to follow the pattern: get rid of custom inits, take `uiConfig` from the responder chain, etc.

Reply:
Thank you, I removed the uiConfig passing on init, the lest I could do. Next steps will probably be removing the whole inits, right now it would be probably a bit complicated with replacing `channelAndUserId` and the order of calling functions :)